Routing/Loader/DirectoryRouteLoader.php

```diff
@@ -50,8 +50,10 @@ public function load($resource, $type = null)
         $finder = new Finder();
 
         foreach ($finder->in($resource)->name('*.php')->files() as $file) {
-            $imported = $this->processor->importResource($this, ClassUtils::findClassInFile($file), array(), null, null, 'rest');
-            $collection->addCollection($imported);
+            if($class = ClassUtils::findClassInFile($file)) {
```

Comment:
I was thinking about a name such as `0` but i realized that's not allowed so it's fine for me as is :)